packages/gestalt/src/PageHeaderComponents.js

```diff
@@ -0,0 +1,212 @@
+      </Mask>
+    </Box>
+  );
+}
+
+export function PageHeaderBadge({
+  badgeText,
+  badgeTooltipText,
+}: {|
+  badgeText: string,
+  badgeTooltipText?: string,
+|}): Node {
+  return (
+    <Fragment>
+      {badgeText && !badgeTooltipText ? (
+        <Badge text={badgeText} type="info" position="top" />
+      ) : null}
+      {badgeText && badgeTooltipText ? (
+        <Tooltip accessibilityLabel="" text={badgeTooltipText} idealDirection="up">
+          <Badge text={badgeText} type="info" position="top" />
+        </Tooltip>
+      ) : null}
+    </Fragment>
+  );
+}
+
+export function PageHeaderHelperIconButton({
+  accessibilityLabel,
+  accessibilityControls,
+  accessibilityExpanded,
+  onClick,
+}: {|
+  accessibilityLabel: string,
+  accessibilityControls: string,
+  accessibilityExpanded: boolean,
+  onClick: ({|
+    event:
+      | SyntheticMouseEvent<HTMLButtonElement>
+      | SyntheticKeyboardEvent<HTMLButtonElement>
+      | SyntheticMouseEvent<HTMLAnchorElement>
+      | SyntheticKeyboardEvent<HTMLAnchorElement>,
+    dangerouslyDisableOnNavigation: () => void,
+  |}) => void,
+|}): Node {
+  return (
+    <Box dangerouslySetInlineStyle={{ __style: { marginTop: '-1px' } }} rounding="circle">
+      <IconButton
+        accessibilityControls={accessibilityControls}
+        accessibilityExpanded={accessibilityExpanded}
+        accessibilityHaspopup
+        accessibilityLabel={accessibilityLabel}
+        bgColor="lightGray"
+        icon="question-mark"
+        iconColor="darkGray"
+        onClick={onClick}
+        size="xs"
+      />
+    </Box>
+  );
+}
+
+export function PageHeaderSubtext({
+  subtext,
+  helperLink,
+}: {|
+  subtext: string,
+  helperLink?: {|
+    text: string,
+    accessibilityLabel: string,
+    href: string,
+    onClick: ({|
+      event: SyntheticMouseEvent<HTMLAnchorElement> | SyntheticKeyboardEvent<HTMLAnchorElement>,
+      dangerouslyDisableOnNavigation: () => void,
+    |}) => void,
+  |},
+|}): Node {
+  return (
+    <Box display="none" smDisplay="block">
+      <Text inline>
+        {subtext}
+        {helperLink ? (
+          <Fragment>
+            {' '}
+            <Text inline underline>
+              <Link
+                accessibilityLabel={helperLink.accessibilityLabel}
+                href={helperLink.href}
+                onClick={helperLink.onClick}
+                target="blank"
+                inline
+              >
+                <span style={{ textDecoration: 'underline' }}>{helperLink.text}</span>
+              </Link>
+            </Text>
+          </Fragment>
+        ) : null}
+      </Text>
+    </Box>
+  );
+}
+
+export function PageHeaderActionBlock({
+  primaryAction,
+  secondaryAction,
+  dropdownAccessibilityLabel = '',
+}: {|
+  primaryAction: {| component: ActionType, dropdownItems: $ReadOnlyArray<Node> |},
+  secondaryAction?: {| component: ActionType, dropdownItems: $ReadOnlyArray<Node> |},
+  dropdownAccessibilityLabel?: string,
```

Comment:
Can we add an error check somewhere to ensure that the user always passes dropdownAccessibilityLabel when actions are provided?

Reply:
I'm gonna add a default `dropdownAccessibilityLabel` with i18n provider, that should cover it. I could triple check in case there's no default, but we don't do that anywhere in the code.